src/frontend/src/components/NavBar/index.tsx

```diff
@@ -1,11 +1,9 @@
-import cx from "classnames";
 import Link from "next/link";
 import React from "react";
 import { useUserInfo } from "src/common/queries/auth";
 import { ROUTES } from "src/common/routes";
 import RightNav from "./components/RightNav";
-import style from "./index.module.scss";
-import { Logo, LogoAnchor } from "./style";
+import { LeftNav, Logo, LogoAnchor, NavBar, NavOrg, Separator } from "./style";
 
 // (thuang): Please make sure this value is in sync with what we have in css
```

Comment:
Neither here nor there about your work, but I think this comment is out of date and no longer matters. Checking the code, appears that `NAV_BAR_HEIGHT_PX` only shows up in one other file, where it's being imported from here.